crates/foundry/forge/src/lib.rs

```diff
@@ -238,4 +291,16 @@ impl TestOptionsBuilder {
         let base_invariant = self.invariant.unwrap_or_default();
         TestOptions::new(output, root, profiles, base_fuzz, base_invariant)
     }
+
+    /// Creates an instance of [`TestOptions`]. This takes care of creating
+    /// "fuzz" and "invariant" fallbacks, and extracting all inline test
+    /// configs, if available.
+    pub fn build_hardhat(self) -> Result<TestOptions, InlineConfigError> {
```

Comment:
How are we currently compiling inline tests?

Reply:
We don't currently, doc is just leftover. I'd keep it like this until the issue is more comprehensively resolved in the follow up